### PR TITLE
feat(platforms): moves procedures to Platforms/src/Twitter

### PR DESCRIPTION
**Pull request by farque65**

### CODE REVIEW

File: platforms/src/Twitter/procedures/index.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import * as twitterOAuth from "./twitterOauth";
+
+export const router = Router();
+
+export type GenerateTwitterAuthUrlRequestBody = {
+  callback: string;
+};
+
+export type GenerateBrightidBody = {
```
Comment (schultztimothy) [inline platforms/src/Twitter/procedures/index.ts:12]: Looks like this might have been moved over accidentally

**Review by schultztimothy — APPROVED** [review]
lgtm 🚀

File: platforms/src/Twitter/procedures/index.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
+// ---- Server
+import { Request, Response, Router } from "express";
+
+import * as twitterOAuth from "./twitterOauth";
+import { triggerBrightidSponsorship, verifyBrightidContextId } from "./brightid";
```
Comment (aminah-io) [inline platforms/src/Twitter/procedures/index.ts:5]: Same question as above--should BrightID be under the Twitter platform?

**Review by aminah-io — COMMENTED** [review]
Couple of questions about BrightID being under the Twitter platform 👏

File: platforms/src/Twitter/procedures/brightid.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import { verifyContextId, sponsor } from "brightid_sdk";
```
[the remark this answers is not in the document]
Comment (farque65) [reply platforms/src/Twitter/procedures/brightid.ts:1]: Yes, Bright Id should be moved out of the twitter platform. There needs to be one file defining the router that will be imported by the IAM server. I will move the routing file to a top level file called procedureRouter.

**Review by aminah-io — APPROVED** [review]
LGTM 🚀

**Pull request merged**